goldmane/fv/daemon_test.go

```diff
@@ -290,3 +290,72 @@ func TestHints(t *testing.T) {
 		return nil
 	}, 5*time.Second, 1*time.Second).Should(Succeed())
 }
+
+func TestStatistics(t *testing.T) {
+	cfg := daemon.Config{
+		LogLevel:                 "debug",
+		Port:                     8988,
+		AggregationWindow:        time.Second * 1,
+		EmitAfterSeconds:         2,
+		EmitterAggregationWindow: time.Second * 2,
+	}
+	defer daemonSetup(t, cfg)()
```

Comment:
`createKeyCertPair()` or `daemonSetup()` look like utility functions. Should we add `t.Helper()` into these functions to make stack traces clearer when they fail. Maybe we should also think about moving them to some utility class to make test code cleaner and improve reusability.

Reply:
>  about moving them to some utility class to make test code cleaner and improve reusability.

Maybe? they are already reused within this test suite. If we find a spot that can use them sure, but we should be careful about over engineering utilities. It's not always better to write something as a library function if it's actually only used in one place!

Better to keep it private and then export it later if we end up needing it somewhere else.